test/module/irohad/torii/cache/cache_test.cpp

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "torii/cache/cache.hpp"
+#include <endpoint.pb.h>
+#include <gtest/gtest.h>
+
+using namespace torii::cache;
+using namespace iroha::protocol;
+
+const int typicalInsertAmount = 5;
```

Comment:
What is variable means?

Reply:
It's just used as a limit for for loops without any deeper meaning.